docs/OV_Runtime_UG/Samples_Overview.md

```diff
@@ -128,55 +131,56 @@ cd build
 make
 ```
 
-For the release configuration, the sample application binaries are in `<path_to_build_directory>/intel64/Release/`;
-for the debug configuration — in `<path_to_build_directory>/intel64/Debug/`.
+The sample application binaries can be found in:
+* The release configuration: `<path_to_build_directory>/intel64/Release/`,
+* The debug configuration - `<path_to_build_directory>/intel64/Debug/`.
 
 ### <a name="build_samples_windows"></a>Build the Sample Applications on Microsoft Windows
 
 The recommended Windows build environment is the following:
-* Microsoft Windows 10
-* Microsoft Visual Studio 2019
-* CMake version 3.10 or higher
+* Microsoft Windows 10,
+* Microsoft Visual Studio 2019,
+* CMake version 3.10 or higher.
 
-> **NOTE**: If you want to use Microsoft Visual Studio 2019, you are required to install CMake 3.14 or higher.
+> **NOTE**: To use Microsoft Visual Studio 2019, it is required to install CMake 3.14 or higher.
 
 To build the C or C++ sample applications on Windows, go to the `<INSTALL_DIR>\samples\c` or `<INSTALL_DIR>\samples\cpp` directory, respectively, and run the `build_samples_msvc.bat` batch file:
 ```sh
 build_samples_msvc.bat
 ```
 
-By default, the script automatically detects the highest Microsoft Visual Studio version installed on the machine and uses it to create and build a solution for a sample code
+By default, the script automatically detects the newest Microsoft Visual Studio version installed on the machine and uses it to build a solution for a sample code
```

Comment:
```suggestion
By default, the script automatically detects the highest Microsoft Visual Studio version installed on the machine and uses it to build a solution for a sample code.
```
I'd leave the 'highest' as it's safest approach, while it is possible to install MS VS versions side-by-side.